src/pages/waiting-room/cat-be/waiting-room.cat-be.page.ts

```diff
@@ -101,6 +103,10 @@ export class WaitingRoomCatBEPage extends BasePageComponent implements OnInit {
     if (super.isIos()) {
       this.screenOrientation.lock(this.screenOrientation.ORIENTATIONS.PORTRAIT_PRIMARY);
       this.insomnia.keepAwake();
+
+      if (!this.isPracticeMode) {
```

Comment:
Should have mentioned this earlier, I don't think B+E is a practice-able test. So we can remove this if statement and just enable single app mode when the platform is iOS.

Reply:
Ah, ok. So is CAT B going to be the only category that will be practice-able?

Reply:
Yeah, I believe so. :)

Reply:
yes

Reply:
Cheers guys, good to know for when we implement this fix for CAT C 👍🏾